code/game/objects/items/food/sweets.dm

```diff
@@ -232,7 +232,6 @@
 	metabolization_amount = REAGENTS_METABOLISM
 
 /obj/item/food/bubblegum/bubblegum/process()
-	. = ..()
```

Comment:
From what I can tell this item doesn't even *start* processing, so this code is broken in two ways

Also `/obj/item/clothing/suit/hooded/hostile_environment/process` does the same thing, so feel free to check that out too

Reply:
Done